conversation_event.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type EventParams struct {
+	// Type identifies the type of event (start-typing, stop-typing, joined, left)
+	Type string `json:"name"`
+
+	// ParticipantID identifies the message sender.
+	ParticipantID string `json:"participant_id"`
+
+	// ParticipantType identifies the type of participant who sent this message.
+	ParticipantType ParticipantType `json:"participant_type"`
+
+	// Timestamp optionally defines the time when the conversation was assigned.
+	// If not given, this will default to the current time.
+	Timestamp *time.Time `json:"timestamp,omitempty"`
+}
```

Comment:
I forgot to mention this in the Slack thread, but could we add an optional `IdempotencyKey` field so folks can retry the request 🙏

Reply:
✅